custom_components/mikrotik_router/mikrotik_controller.py

```diff
@@ -3,6 +3,7 @@
 import asyncio
 import logging
 from datetime import timedelta
+import ipaddress
```

Comment:
Do we need full module?

Reply:
only ipaddress.IPv4Network and ipaddress.ip_address are used. 
we choudl use "from ipaddress import IPv4Network, ip_address"